hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/handlers/ConsensusUpdateTopicHandlerTest.java

```diff
@@ -329,64 +351,67 @@ void appliesNewAutoRenewNumViaMeta() {
     }
 
     @Test
+    @DisplayName("Topic is not updated if no fields are changed")
     void nothingHappensIfUpdateIsNoop() {
         refreshStoresWithCurrentTopicInBothReadableAndWritable();
 
         given(handleContext.expiryValidator()).willReturn(expiryValidator);
         given(handleContext.attributeValidator()).willReturn(attributeValidator);
         // No-op
-        final var op = OP_BUILDER.topicID(wellKnownId()).build();
+        final var op = OP_BUILDER.topicID(topicId).build();
 
         subject.handle(handleContext, op, writableStore);
 
         final var newTopic = writableTopicState.get(topicEntityNum);
         assertEquals(topic, newTopic);
     }
 
-    public static void assertFailsWith(final ResponseCodeEnum status, final Runnable something) {
-        final var ex = assertThrows(HandleException.class, something::run);
-        assertEquals(status, ex.getStatus());
-    }
-
     @Test
+    @DisplayName("Validate Mutate NonExpiryField memo as expected")
```

Comment:
Quite a few more display names here that don't add any information to the person reading the code

Reply:
This is just to test one of || conditions in wantsToMutateNonExpiryField(). It's hard to put a real meaningful name.

Reply:
I updated the display. Please see if it makes more sense to you. I tried to make sure every unit test method has @DisplayName.